src/meshcat/geometry.py

```diff
@@ -195,6 +215,26 @@ def lower(self, object_data):
         }
 
 
+class TextTexture(Texture):
+    def __init__(self, text, font_size=100, font_face='sans-serif',
+                 width=200, height=100, position=[10, 10]):
```

Comment:
`width`, `height`, `position` are not used. Should I delete them ?

Reply:
yes please